src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSource.cs

```diff
@@ -620,87 +649,81 @@ private static SuggestedActionSetPriority GetSuggestedActionSetPriority(CodeActi
                 }
 
                 return new SuggestedActionSet(
-                    refactoringSuggestedActions.ToImmutableAndFree(),
-                    SuggestedActionSetPriority.Low,
-                    applicableSpan);
+                    categoryName: PredefinedSuggestedActionCategoryNames.Refactoring,
```

Comment:
The only constructor that takes a category has an optional title as its third parameter, so we need some of these.